clients/go/pkg/worker/exponentialBackoffSupplier_test.go

```diff
@@ -103,7 +103,7 @@ func TestExponentialBackoffSupplier_ShouldBeRandomizedWithJitter(t *testing.T) {
 	// then
 	for i, delay := range retryDelays {
 		// retryDelay is in bounds
-		betweenBounds := delay > lowerMaxBoundDuration && delay < upperMaxBoundDuration
+		betweenBounds := delay >= lowerMaxBoundDuration && delay <= upperMaxBoundDuration
```

Comment:
I hit the edge-case of it being equal 😅 as jitter is computed as `return (e.random.Float64() * (maxFactor - minFactor)) + minFactor`

Reply:
Good finding!